src/app-text/components/controls/km/km-validation-errors.json

```
@@ -0,0 +1,11 @@
+{
+  "messages":{
+    "Error.Mandatory": "mandatory",
+    "Error.MandatoryAnyOf": "mandatoryAnyOf",
+    "Error.InvalidValue": "invalidValue",
+    "Error.InvalidProperty": "invalidProperty",
+    "Error.UnspecifiedLocale": "unspecifiedLocale",
+    "Error.UnexpectedTerm": "unexpectedTerm",
+    "Error.InvalidType": "invalidType"
+  }
```

```suggestion
    "Error.Mandatory": "mandatory",
    "Error.MandatoryAnyOf": "mandatoryAnyOf",
    "Error.InvalidValue": "invalidValue",
    "Error.InvalidProperty": "invalidProperty",
    "Error.UnspecifiedLocale": "unspecifiedLocale",
    "Error.UnexpectedTerm": "unexpectedTerm",
    "Error.InvalidType": "invalidType"
```
the expectation here  is translation text not the term.